app/views/controllers/field_slips/show.html.erb

```diff
@@ -1,10 +1,12 @@
-<% add_page_title(@field_slip.code) %>
+<% add_page_title("#{:FIELD_SLIP.t}: #{@field_slip.code}") %>
 
 <% if notice %>
   <p class="alert alert-success"><%= notice %></p>
 <% end %>
 
-<h1><%= "#{:FIELD_SLIP.t}: #{@field_slip.code}" %></h1>
+<div>
+  <%= @field_slip.qr_code(field_slip_url(@field_slip)).html_safe %>
+</div>
```

Comment:
You probably will need `html_safe` if it's rendering as `svg`... that's potentially unsafe html.

Reply:
Thanks!